Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1306,6 +1317,10 @@ public function reflectClassProperty($className, PropertyReflection $property)
         $visibility = $property->isPublic() ? self::VISIBILITY_PUBLIC : ($property->isProtected() ? self::VISIBILITY_PROTECTED : self::VISIBILITY_PRIVATE);
         $this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES][$propertyName][self::DATA_PROPERTY_VISIBILITY] = $visibility;
 
+        if ($property->hasType()) {
+            $this->classReflectionData[$className][self::DATA_CLASS_PROPERTIES][$propertyName][self::DATA_PROPERTY_TYPE] = trim((string)$property->getType(), '?');
```

Comment:
Question: Should we check this preemptively against a potential value from the "TagValues" below, which contain the `@var` tag? Only valid edge-case I can think of is the tag containing a concrete class and the property being typed against an interface (or vice-versa if that makes any sense?), so we'd need to check `class_implements()` (and maybe `class_parents()`?). Maybe something for another PR